cypress/component/QueryForm.cy.tsx

```diff
@@ -125,7 +131,8 @@ describe('QueryForm', () => {
         sex={props.sex}
         diagnosis={props.diagnosis}
         isControl={props.isControl}
-        minNumSessions={props.minNumSessions}
+        minNumImagingSessions={props.minNumImagingSessions}
+        minNumPhenotypicSessions={props.minNumPhenotypicSessions}
         setIsControl={props.setIsControl}
```

Comment:
I know this isn't new but the `cy.mount` part is the same for every test. Mind moving it into a `cy.beforeEach` to save on visual space?

Reply:
Don't think we can do that because some tests in there have a `spy` that's passed to the component as a prop during mount.